common/remote/hooks/azure/azurefile.cpp

```diff
@@ -465,6 +465,7 @@ AzureFile::AzureFile(const char *_azureFileName) : fullName(_azureFileName)
             throw makeStringExceptionV(99, "Storage api for plane %s is not azureblob", planeName.str());
 
         unsigned numDevices = plane->getPropInt("@numDevices", 1);
+        IPropertyTree * storageapi = plane->queryPropTree("storageapi");
```

Comment:
picky:  Although this can never be null (because line 460 succeeded) a static code checker could not tell that.

Slightly better would be to move it above line 459:

```
      IPropertyTree * storageapi = plane->queryPropTree("storageapi");
      if (!storageapi)
          throw makeStringExceptionV(99, "No storage api defined for plane %s", planeName.str());
       const char * api = storageapi->queryProp("@type");
        if (!api)
            throw makeStringExceptionV(99, "No storage api type defined for plane %s", planeName.str());
```